README.md

```diff
@@ -1,8 +1,10 @@
 # Budget App
-> Budget App
```

Comment:
- [ ] Good job adding the unit tests for the models. However, you are still missing integration tests for the application. Kindly implement those also.